UnityProject/Assets/Scripts/Objects/Machines/Jukebox.cs

```diff
@@ -320,5 +327,64 @@ public void StateUpdate(PowerState State)
 		}
 
 		#endregion
+
+		public bool WillInteract(HandApply interaction, NetworkSide side)
+		{
+			if (DefaultWillInteract.Default(interaction, side) == false) return false;
+			if (interaction.TargetObject != gameObject) return false;
```

Comment:
this check seems unnecessary, since TargetObject is always going to be the `HandApply` target

Reply:
from what I remember, the interaction framework is silly like that

Reply:
It used to be possible to trigger handapply on an object thats not the target, I dunno if thats still the case though